providers/ProgressProvider.tsx

```diff
@@ -30,12 +30,14 @@ export default function ProgressProvider({
     defaultProgressContext.progress
   )
   const [isLoading, setIsLoading] = useState(true)
-  const [savePending, setSavePending] = useState(false)
 
   const init = async () => {
     try {
       setIsLoading(true)
-      const progress = await getProgress()
+      let progress = await getProgress()
+      if (progress === 'CH1INT1') {
```

Comment:
Nitpick but could we get `CH1INT1` from `lib/progress` instead?

Reply:
Yes that, or export a new variable, as long as we can edit from the single file `lib/progress.ts` as opposed to going through the codebase to find all occurences of `CH1INT1` if we ever change it.